book/website/reproducible-research/rdm/rdm-spreadsheets.md

```diff
@@ -1,35 +1,134 @@
+If you are using excel in a team, be sure to all use the same version, as the default decimal value may change.
+
+Finally, restricting values in specific columns (linking spreadsheets with taxonomies or ontologies) can be very useful. 
+There are specific tools to help people fill such columns, as well as tools to validate the spreadsheets before integration in the analysis.
+
+### Standard and versioning
+
+A good spreadsheet design has instructive and intuitive header names, and it facilitates both data collection and analysis. 
+Building such a spreadsheet design is difficult, takes time, iteration and consensus. It is therefore very useful to look for standard spreadsheet before designing your own, and share your design openly once you created yours. 
+One should also use a version history of the spreadsheets (as they will evolve over time) and analysis script should mention the version number of the spreadsheet. 
+Documentation of the spreadsheet, its version history, and the ontologies it is linked to, can be useful for future users. 
+
+### Working in team: wrap-up
+
+If you are working with a team on data collection, make sure:
+- everyone is using the same software (and software version) to enter the data.
+- everyone is using the same version of the spreadsheet template.
+- everyone understand what each column represents, and the unit that should be used.
+- every column has a defined standard on how to enter data in it, or a taxonomy of terms that one can use.
+- One person is responsible to answer putative questions during data collection.
+- every spreadsheet is validated before entering the analysis workflow, and as soon as possible.
+
+## Data manipulation and analysis
+
+**Do not do data manipulation or analysis in a spreadsheet program.**
+
+In particular, only copy-paste from one spreadsheet to another if the process is used very rarely. 
+It is now very easy to read and combine the different spreadsheets in the analysis software, with the additional advantage that the software will return an error message if the headers do not fit.
+
+
+
+
+
+## Summary
+
+While spreadsheet can be a very user friendly way to collect and share data, they can be the source of mistakes if used inappropriately. When aiming at doing reproducible analyses, one should design the spreadsheet for computer-readability and easy data analysis before starting data collection. Data manipulation and analysis in spreadsheets, in particular, is best avoided as it lead to non-reproducible workflows. Using version control and making the data read-only are two additional data management practices that can prevent accidents.
+
+
+Use a README texte file to explain naming conventions on top of the spreadsheet desing choices, so that it will also become clear to others what the file names mean. If you work in team, you should take particular care to the rules and make sure everyone is following them. 
```

Comment:
```suggestion
Use a README file to explain naming conventions and other design choices on top of the spreadsheet.
It will make it clear to others what the file names mean and what criteria to consider when designing an analysis workflow. 
If you work in a team, you should take particular care of the conventions and make sure everyone follows them. 
```